docs/CollectionsApi.md

```diff
@@ -17,7 +17,7 @@ All URIs are relative to *https://api.search.io*
 
 ## create_collection
 
-> <Collection> create_collection(collection_id, collection)
+> <Collection> create_collection(collection_id, collection, opts)
```

Comment:
@jonathaningram this an incompatible API change?

Reply:
@jkaho I don't think so. See https://github.com/sajari/sdk_ruby/pull/4/files?show-viewed-files=true&file-filters%5B%5D=#diff-15a79a26c88566b49854a47c39549808fddec67a725a79fb32866dc5a12a70bfR29

The `opts` have a default value. Would still be a minor version bump though.

Reply:
@jonathaningram ah okay. Which file/line does that link to? I'm getting `There are no files selected for viewing`

Reply:
@jkaho lib/sdk_ruby/api/collections_api.rb:29